### PR TITLE
[DEV-2865], [DEV-2791], [NO-TASK]

### PR DESCRIPTION
**Pull request by lapatric**
- [DEV-2865] move infoText into dropdown in StyledDataTableExpandableRow
- [NO-TASK] fix text overflow issues in StyledDropdown & StyledDropdownMultiChoice
- [NO-TASK] fix hover color confusion in StyledDropdown & StyledDropdownMultiChoice
- [DEV-2791] update delete account endpoint + fix function signature

[DEV-2865]: https://dfxswiss.atlassian.net/browse/DEV-2865?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[DEV-2791]: https://dfxswiss.atlassian.net/browse/DEV-2791?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/react/src/definitions/user.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { TransactionFilterKey } from './transaction';
 export const UserUrl = {
   get: 'user',
   ref: 'user/ref',
-  delete: 'user',
+  delete: 'user/account',
```
Comment (davidleomay) [inline packages/react/src/definitions/user.ts]: Is this right?

**Review by davidleomay — APPROVED** [review]

File: packages/react/src/definitions/user.ts (hunk 1 above)
Comment (lapatric) [reply packages/react/src/definitions/user.ts]: Should be correct I think 🤔

- [Here is the API endpoint: deleteAccount](https://github.com/DFXswiss/api/blob/f25394de0ad7680e029c0b2dcd9697eb02520196/src/subdomains/generic/user/models/user/user.controller.ts#L254)
- [And this is how it's used in the user hook](https://github.com/DFXswiss/packages/blob/1a80a482c7b9792394aa87c60ac4be5ea9b48583/packages/react/src/hooks/user.hook.ts#L73)

Comment (davidleomay) [reply packages/react/src/definitions/user.ts]: Ah yes, just saw it. But wouldn't it make sense to have it on `DELETE user` in the backend? Would be a better fit to the rest of the paths.

Comment (lapatric) [reply packages/react/src/definitions/user.ts]: I'm good with that too 👍 @Yannick1712? (We had changed it to match the deprecated delete account endpoint if I recall correctly)

Comment (Yannick1712) [reply packages/react/src/definitions/user.ts]: Yes that's right @lapatric we changed that 👍 

@davidleomay Hm your deactivating the userAccount (userData) with that user endpoint. For me it's a bit strange to have a `Delete()` on user controller, which "deletes"/deactivates not the user but the userData instead 🤔 that's why for me `Delete(account)` makes more sense in the user controller. 
But if both of you think that `Delete()` would be better, I'm fine with that ✌️

Comment (davidleomay) [reply packages/react/src/definitions/user.ts]: @lapatric let's change it to `user` only 😉

**Review by davidleomay — APPROVED** [review]

**Pull request merged**